pkg/container-hook/bpf/execruntime.bpf.c

```diff
@@ -31,10 +31,44 @@ struct {
 struct {
 	__uint(type, BPF_MAP_TYPE_HASH);
 	__uint(max_entries, 128);
-	__type(key, u32); // tgid (fanotify will need to lookup by tgid)
+	__type(key, u32); // tid
```

Comment:
| Kernel terminology   | Userspace terminology |
|----------------------|-----------------------|
| tgid (task group) | pid (process id)      |
| pid     | tid (thread id)       |

Should I use the kernel terminology in the code everywhere?

In the kernel terminology, this `exec_args` hash map was keyed by a tgid before the patch and I changed it to be keyed by a pid in order to fix the leak.

It makes things slightly more complicated in `kretprobe/fsnotify_remove_first_event` because the fanotify_event contains the tgid. So we have to use the `pid_by_tgid` map to find the pid, so we can do the lookup in `exec_args`.

Reply:
> Should I use the kernel terminology in the code everywhere?

I think it's a good idea to use kernel terminology on eBPF programs.